kotlinx-coroutines-test/build.gradle.kts

```diff
@@ -27,4 +27,14 @@ kotlin {
             }
         }
     }
-}
+
+    wasm {
+        nodejs {
+            testTask {
+                filter.apply {
+                    excludeTest("TestDispatchersTest", "testMainMocking")
```

Comment:
Please leave a comment why the test is ignored

Reply:
Done